pkg/middleware/auth.go

```diff
@@ -130,15 +131,15 @@ func (c *JWTVerifierURL) updateKeySet() error {
 	c.jwksLock.Lock()
 	defer c.jwksLock.Unlock()
 
-	set, err := jwk.Fetch(c.jwksURL)
+	set, err := jwk.Fetch(context.TODO(), c.jwksURL)
```

Comment:
I'm totally not knowledgeable, but `.TODO` caught my eye. This is correct right?

Reply:
Yes, for now, we have to pass a context there. TODO() is for in future we should pass the caller context here if possible.